bw2analyzer/lci.py

```diff
@@ -1,14 +1,60 @@
 import pandas as pd
-from bw2data import get_activity
+from bw2data import Database
 from bw2calc import LCA
+from typing import List, Union
 
 
-def get_labeled_inventory(lca: LCA) -> pd.DataFrame:
+def _get_dependent_database_names(lca: LCA) -> List[str]:
+    databases = []
+    for key in lca.remapping_dicts.keys():
+        databases += [key[0] for key in lca.remapping_dicts[key].values()]
+    return pd.unique(databases)
+
+
+def _load_database_metadata(
+    lca: LCA, cols: Union[None, List[str], str] = None
+) -> pd.DataFrame:
+    # get database names
+    databases = _get_dependent_database_names(lca)
+    # load metadata and store in one dataframe
+    df = pd.DataFrame()
+    for db in databases:
+        df = df.append(pd.DataFrame(Database(db)), ignore_index=True)
+    # drop unwanted columns
+    if cols is None:
+        cols = [
+            "id",
+            "name",
+            "location",
+            "reference product",
+            "type",
+            "unit",
+            "database",
+            "categories",
+        ]
+    elif cols == "all":
+        cols = df.columns.tolist()
+    existing_cols = [c for c in cols if c in df]
+    df = df[existing_cols]
+    # set index
```

Comment:
Maybe unnecessary :)

Reply:
Only unnecessary if `usecols = 'all'`. In all other cases, it's necessary.

Reply:
Sorry, I meant the comment :)

Reply:
Oh! Hahaha, true.